kie-api/src/build/revapi-config.json

```diff
@@ -307,6 +307,16 @@
        "code": "java.method.addedToInterface",
        "new": "method <T> T org.kie.api.runtime.KieSession::getKieRuntime(java.lang.Class<T>)",
        "justification": "Promoting internal method getKieRuntime() to the public API"
+      },
+      {
+        "code": "java.method.removed",
+        "old": "method java.util.List<org.kie.api.task.model.TaskSummary> org.kie.api.task.TaskService::getTasksByVariousFields(java.lang.String, java.util.List<java.lang.Long>, java.util.List<java.lang.Long>, java.util.List<java.lang.Long>, java.util.List<java.lang.String>, java.util.List<java.lang.String>, java.util.List<java.lang.String>, java.util.List<org.kie.api.task.model.Status>, java.util.List<java.lang.String>, boolean)",
+        "justification": "Deprecated"
```

Comment:
Let's change this to something like "Removing API previously deprecated." Sentences generally look and work better for communicating things.